automatic/1password/update.ps1

```diff
@@ -53,18 +53,22 @@ function Get-LatestOPW {
   }
 }
 
-$releases_opw4 = 'https://app-updates.agilebits.com/download/OPW4'
-$kind_opw4 = '1password4'
-$releases_opw = 'https://app-updates.agilebits.com/download/OPW7/Y'
-$kind_opw = '1password'
+$releases = 'https://downloads.1password.com/win/1PasswordSetup-8.8.0.exe'
 
 function global:au_GetLatest {
-  $streams = [ordered] @{
-    OPW4 = Get-LatestOPW -url $releases_opw4 -kind $kind_opw4
-    OPW  = Get-LatestOPW -url $releases_opw -kind $kind_opw
-  }
+  $destination = Join-Path -Path $env:TEMP -ChildPath '1PasswordSetup-8.8.0.exe'
+  Invoke-WebRequest -Uri $releases -UseBasicParsing -OutFile $destination
+
+  (Get-ItemProperty $destination).VersionInfo.ProductVersion
 
-  return @{ Streams = $streams }
+  $version = Get-Item $destination | ForEach-Object { [System.Diagnostics.FileVersionInfo]::GetVersionInfo($_).FileVersion }
+  $checksum = Get-FileHash $destination | ForEach-Object Hash
+  Remove-Item $destination -ErrorAction:Stop
+  @{
+      Version     = $version
+      URL         = $releases
+      Checksum    = $checksum
+  }
 }
 
-update -ChecksumFor 32 -IncludeStream $IncludeStream -Force:$Force
+update -ChecksumFor 32 -Force:$Force
```

Comment:
Did you checked my initial implementation? Did you checked the comments from @crosscontrol?

My previous implementation tried to avoid the card coding of the version, but it was turn down by @crosscontrol. 

So far, I'm pretty much certain that **there is no way how to get the version otherwise than download the package**. 

We have to settle on the approach here, otherwise will be going back and forth.

Reply:
I had not checked your initial implementation nor the comments made previously.

The user is both correct and incorrect at the same time.

The correct part is that you should not use the `-latest` URL (since it does not redirect), and would prevent older versions of the package from working.

The user is, however, incorrect about changing it to a hard-coded URL, and just to note, your initial implementation would have been rejected as it removes the checksum.

If you haven't already, check the comment made by @jacktose in #1773#issuecomment-1244674951 that mentions a way to get the version. Using that information you can build the URL dynamically.

Lastly, just to note. The user @crosscontrol is not a full member of the repository, but rather a community member (same as you) and don't really have the ability to turn down any changes (we still do appreciate efforts by community members by making suggestions though, which unfortunately this time was not correct).

Reply:
One other note, @crosscontrol never purported to be a member or tried to reject anything but instead is just trying to make suggestions to help move this along, in support of another community member making a good effort.

In any case, I greatly appreciate you getting involved as this and your expertise will help a lot of frustrated people once it's figured out! Thank you for the help @AdmiringWorm.

Reply:
To wit, my suggestion was to scrape https://app-updates.agilebits.com/product_history/OPW8 for the first listing without `.NIGHTLY`